test/serialisers/json-0.6.js

```diff
@@ -883,9 +883,9 @@ describe('JSON 0.6 Serialiser', function() {
       });
 
       var metadata = category.attributes.get('metadata');
-      expect(metadata).to.be.instanceof(minim.elements.Object);
+      expect(metadata).to.be.instanceof(minim.elements.Array);
 
-      var member = metadata.getMember('HOST');
+      var member = metadata.get(0);
```

Comment:
This are some of the reasons it should be an object element because it contains members and array elements API doesn't offer any ability to deal with them. It is to late to make this change though.